src/Paket.Core/Cultures.fs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Paket
+open System
+open System.Collections.Generic
+open System.Globalization
+
+[<CompilationRepresentation (CompilationRepresentationFlags.ModuleSuffix)>]
```

Comment:
That one is a bit tricky. not sure if my fix was good. have to think about it a bit more.